ha/monitor/k8s/object_monitor.py

```diff
@@ -82,6 +88,31 @@ def handle_sigterm(self, k8s_watch: watch.Watch, k8s_watch_stream):
         Log.info(f"Stopped watching for {self._object} events.")
         self._stop_event_processing = True
 
+    def is_publish_enable(self) -> bool:
+        """
+        Check if cluster_stop key is exist and if exist check if it is enable
+        if enable then publish event should be stopped
+        """
+        if self._publish_alert:
+            if self._confstore.key_exists(const.CLUSTER_STOP_KEY):
+                _, cluster_stop = (self._confstore.get(const.CLUSTER_STOP_KEY)).popitem()
+                if cluster_stop == const.CLUSTER_STOP_VAL_ENABLE:
+                    Log.info(f"{self._object}_monitor stopping publish alert as cluster stop message is received.")
+                    self._publish_alert = False
+        return self._publish_alert
+
+    def publish_alert(self, alert):
+        """
+        If publish is enabled, i.e. Yet the cluster stop message has not been received.
+        then publish the alert for Fault Tolerance Monitor
+        """
+        if self.is_publish_enable():
+            # Write to message bus
+            Log.info(f"{self._object}_monitor sending alert on message bus {alert.to_dict()}")
+            self._producer.publish(str(alert.to_dict()))
+        else:
+            Log.debug(f"{self._object}_monitor skipping publish for alert: {str(alert.to_dict())}.")
```

Comment:
Please modify the log to indicate publish skipped since cluster stop received

Reply:
done